engine/convert.go

```diff
@@ -117,6 +117,19 @@ func toVolumes(spec *Spec) []v1.Volume {
 			volumes = append(volumes, volume)
 		}
 
+		if v.Claim != nil {
+			volume := v1.Volume{
+				Name: v.Claim.ID,
+				VolumeSource: v1.VolumeSource{
+					PersistentVolumeClaim: &v1.PersistentVolumeClaimVolumeSource{
+						ClaimName: v.Claim.ClaimName,
+						ReadOnly:  true,
```

Comment:
one last question -- should this be hard-coded to readonly or should we provide the option for read write?

Reply:
yeah, we can change this, for our use case is not needed but sounds like a useful configuration.
what about to add the the following to the .drone.yml 
```
volumes:
- name: shared
  claim:
    name: received-data-claim
    read_only: <true|false>
```